content_schemas/formats/shared/definitions/_logo.jsonnet

```diff
@@ -0,0 +1,32 @@
+{
+  type: "object",
+  properties: {
```

Comment:
Should this also have additional properties set to false?

Reply:
Yes, it should.  Have added that in.